src/rl78.c

```diff
@@ -29,7 +31,13 @@ extern int verbose_level;
 static unsigned char communication_mode;
 
 static void rl78_set_reset(port_handle_t fd, int mode, int value)
+// Set reset signal for MCU based on mode (not applicable for mode 5 with hardware switch)
 {
+    if (mode == 4) // operating in mode 5
+    {
+        printf("No GPIO SW reset available, must use switch\r\n");
```

Comment:
Pedantic Pet Peeve of mine: feel free to ignore. ;-)

Switches are latching and/or sliding/rotating, buttons are momentary and/or linear motion.  This is a button, not a switch.

Yes, yes, this depends on how you implement the hardware.  But I've never seen a Reset Switch, only ever Reset Buttons.  Possibly a Reset Momentary Switch on front-panels of old mini-computers.  But these are almost certainly going to be buttons.